recipe/meta.yaml

```diff
@@ -388,7 +384,7 @@ outputs:
         # tools/ is needed to optimise test run
         # as of pytorch=2.0.0, there is a bug when trying to run tests without the tools
         - tools
-        #- .ci/pytorch/smoke_test/smoke_test.py
+        - .ci/pytorch/smoke_test/smoke_test.py
```

Comment:
@danpetry, it seems this file is not part of the tarball:
```
RuntimeError: Did not find any source_files for test with pattern .ci/pytorch/smoke_test/smoke_test.py
```

Reply:
we could also do something like [this](https://github.com/AnacondaRecipes/pytorch-feedstock/blob/4f8a8bbf9cbcf41605ccddceaa1f4151ad609de3/recipe/meta.yaml#L58) to remove the need to manually update. Maybe later.

Reply:
ah, should have checked the anaconda recipe. Fine for me to use the same approach here!